vllm/v1/request.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         self._all_token_ids: list[int] = self.prompt_token_ids.copy()
         self.spec_token_ids: list[int] = []
         self.num_computed_tokens = 0
+        self.num_cached_tokens = 0
```

Comment:
I think this is confusing.

Reply:
@simon-mo Please add a comment. I think people will be confused between `num_computed_tokens` and `num_cached_tokens` otw.